<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>321</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;HY견고딕&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="mem_tab">
    <attribute name="title">
     <string>회원 요금</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>309</width>
       <height>341</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="memBtn_1">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>1,000 원
1:20</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="memBtn_2">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>2,000 원
2:40</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="memBtn_3">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>3,000 원
4:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="memBtn_4">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>5,000 원
7:00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="memBtn_5">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>10,000 원
15:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="memBtn_6">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>30,000 원
50:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="empty_1">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="empty_2">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="temp_mem_tab">
    <attribute name="title">
     <string>비회원 요금</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>309</width>
       <height>340</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="tempBtn_1">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>1,000 원
1:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tempBtn_2">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>2,000 원
2:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tempBtn_3">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>3,000 원
3:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tempBtn_4">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>5,000 원
6:00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="tempBtn_5">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>10,000 원
13:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tempBtn_6">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>30,000 원
45:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="empty_3">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="empty_4">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="paymentBtn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>420</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>결  제</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>회원 찾기</string>
   </property>
   <widget class="QLabel" name="id_result_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>result_label</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>284</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>HY견고딕</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>아이디</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="id_line_edit"/>
     </item>
     <item>
      <widget class="QPushButton" name="id_searchBtn">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>검색</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>311</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>충전 정보 </string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>201</width>
      <height>128</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <property name="spacing">
      <number>30</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>회원 ID   : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="id_label">
           <property name="text">
            <string>temp</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>잔여 시간 : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="remain_label">
           <property name="text">
            <string>00 : 00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>충전 시간 : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="time_label">
           <property name="text">
            <string>00 : 00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>결제 요금 : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="price_label">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0 원</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
